src/metar/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+list(APPEND programs
```

Comment:
Do we expect additional METAR converters? If not, I suggest making the directory more generic, like "conv" or "conventional" and keeping the metar_csv2ioda.py filename.

Reply:
A totally fair comment.  A "conventional" subdir might be a good choice.  Other than a direct BUFR file convert of METAR to IODA via Ron's software, I also doubt there would be multiple converters.  Do you happen to know if I would need to create an entirely new github feature branch if I eliminated the metar subdir in favor of a newly named subdir, or does a folder renaming survive properly under the same branch name?  In old CVS, this would generate problems.

Reply:
As far as I know there is no problem with renaming directories. In fact, in git, if you create a blank directory, add it, but don't put any tracked files in it, it won't even track that the directory exists. So I think you're free to change the directory name in this branch/PR without issues.

Reply:
Cool. Good to know.  I will go ahead with the renaming because I foresee a good likelihood of similar possible converters that I may be contributing for other conventional data prior to usage of Ron's BUFR-IODA direct.  I'm using these CSV-IODA converters to accelerate my work with UFO developments so even if these are quite temporary, I can be highly productive in preparing future UFO work/codes.